packages/lesswrong/components/tagging/TagAudioPlayerWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib/components';
+import { isTagAllowedType3Audio } from '../../lib/collections/tags/helpers';
+
+const styles = (theme: ThemeType) => ({
```

Comment:
Should probably default to `defineStyles` now, and omit the `styles` HoC in favor of `useStyles`